### ISSUE TITLE
The brunch update for r93 version of Chrome OS has not yet been released

### ISSUE THREAD

**Issue by Xerrious** [BiteDasher/brcr-update#8]
Hi,

I'm facing an issue with the script not being able to find the r93 version, when curently brunch is at r94. The strange thing is that earlier today I did this method, using your script and it worked perfectly. It update my system from Chrome OS 93 and Brunch 91, both to 94. For some reason I had to format my drive and make everything again. I've reinstalled Chrome OS, but when I ran ur script again I got this error as you can see in the screenshot. Any ideea what might cause this and why it worked some hours ago and now not? Thank you in advance

![Screenshot 2022-01-04 9 47 48 PM](https://user-images.githubusercontent.com/88162550/148122180-e3b28d6d-29be-46e8-b064-7c0b6db1bd8c.png)

**Comment by Xerrious** [comment]
Hi again,

So I've updated brunch from r91 to r93 using the old method from Sebanc, and I tried again to update r94 using the script. But I still get the same message, even if I now have brunch r93.
![Screenshot 2022-01-04 11 02 38 PM](https://user-images.githubusercontent.com/88162550/148129832-0401438e-b92f-4a95-94b0-22825f971218.png)

**Comment by drjacka** [comment]
I'm getting (The brunch update for r94 version of Chrome OS has not yet been released
). Currently runnig on v94 so I'm guessing it's having issues identifying the latest available version.

**Comment by Xerrious** [comment]
Yes. I tried with 94 also and the same. What's weird is that 1st time (like 4-5 hours before I got this message) it worked fine.

**Comment by ghost** [comment]
still not fixed in r100... :'(
